package.json

```diff
@@ -108,6 +109,7 @@
     "test:command-reference": "./bin/run commandreference:generate --erroronwarnings",
     "test:deprecation-policy": "./bin/run snapshot:compare",
     "test:nuts": "nyc mocha \"**/*.nut.ts\" --slow 4500 --timeout 600000 --parallel",
+    "test:nuts:sandbox": "nyc mocha \"**/*.sandboxNut.ts\" --slow 450000 --timeout 7200000 --parallel",
```

Comment:
we normally don't run the individual nuts with `parallel` unless there's a bunch of them.  It's easier to see the progress as the individual `it` run without this.